manifest_staging/charts/aad-pod-identity/templates/identities.yaml

```diff
@@ -6,7 +6,7 @@ items:
   - apiVersion: "aadpodidentity.k8s.io/v1"
     kind: AzureIdentity
     metadata:
-      name: {{ $key }}
+      name: {{ default $key $value.name }}
       namespace: {{ default $.Release.Namespace .namespace }}
```

Comment:
Shouldn't this also be `{{ default $.Release.Namespace $value.namespace }}`

Reply:
Looks like both work but I can apply your change to make it clearer